### DISCUSSION TITLE
Start Up Confusion

### DISCUSSION THREAD

**Discussion by lrheath29** [joeyagreco/leeger#52] [Q&A] [answered]
Hello! First off, thank you for taking the time to publish this and advertise it. This project is uniquely cool for people like myself, who are just starting to learn to code and very interested in fantasy football. 

However, despite my best efforts to read through your documentation and some experience writing my own python scripts, I do not know how/where to run your code from. I assume I have to make some small changes (adding league ID, changing owner/team names), but I am not quite sure where to start. 

I have access to Juypter and Spyder and I am comfortable using either one. If you could just guide me a little on where to run these codes and in what order I should make the edits and run them in, that would be greatly appreciated. 

Thanks again!

**Comment by joeyagreco** [comment, marked as answer]
Hi there!

Thanks for checking out the project!

As far as running the code, the steps I follow are

1. Make sure you have Python 3.10 (or greater) installed on your machine
2. Create a script that ends in ".py" with the code you want to run
  - Something [like this](https://github.com/joeyagreco/leeger/tree/main/example/league_loader) for league loaders
  - Something [like this](https://github.com/joeyagreco/leeger/tree/main/example/stat) for getting stats
3. Navigate in your terminal to the folder where your ".py" script from step 2 is
4. Run the script in your terminal using something like "py myScriptName.py", "python myScriptName.py", or python3 myScriptName.py"

If you need help getting it to work with Jupyter Notebook or Spyder we can set up a call to work through it together, as I don't typically use those.

As far as making changes like league ID and stuff like that, you'll need to grab the info for your specific fantasy site listed under "Setup Documentation" [here](https://github.com/joeyagreco/leeger#supported-league-loaders).

Then find your league loader example template [here ](https://github.com/joeyagreco/leeger/tree/main/example/league_loader)and fill it in with your specific info.
